config/typo3.rules

```diff
@@ -0,0 +1,69 @@
+# Harden the `chmod` function
+sp.disable_function.function("chmod").param("mode").filename_r("typo3/sysext/core/Classes/Utility/GeneralUtility.php$").value_r("^[0-9]{2}6$").allow();
+sp.disable_function.function("chmod").param("mode").value_r("^[0-9]{2}[67]$").drop();
+sp.disable_function.function("chmod").param("mode").value_r("o\\+w$").drop();
+
+# Prevent various `mail`-related vulnerabilities
+sp.disable_function.function("mail").param("additional_parameters").value_r("\\-").drop();
```

Comment:
Is the parameter `additional_parameters` even used? If not, we add a rule for anything but an empty string.

Reply:
Please add filename here too, if the `mail` function is used with an additional argument. If it isn't, please blacklist anything but an empty string.